WooCommerce/Classes/Extensions/WKWebView+Authenticated.swift

```diff
@@ -37,8 +37,7 @@ extension WKWebView {
     }
 
     private func authenticatedPostData(with credentials: Credentials?, redirectTo url: URL) throws -> URLRequest {
-        guard let username = credentials?.username,
-              let token = credentials?.authToken else {
+        guard case let .wpcom(username: username, authToken: token, siteAddress: _) = credentials else {
```

Comment:
Nit: this can be simplified:

```suggestion
        guard case let .wpcom(username: username, authToken: token, _) = credentials else {
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/8446/commits/b231bcef7029ee63fab0c1e723c9e645ed2e16e1